src/java/voldemort/server/protocol/admin/AdminServiceRequestHandler.java

```diff
@@ -1054,14 +1054,12 @@ private String swapStore(String storeName, String directory) throws VoldemortExc
                                                                                                             .setStatus("started");
         try {
             if(!metadataStore.getReadOnlyFetchEnabledUnlocked()) {
-                throw new ReadOnlyFetchDisabledException("Read-only fetcher is disabled in "
-                                             + metadataStore.getServerStateUnlocked()
-                                             + " state on node " + metadataStore.getNodeId());
+                throw new ReadOnlyFetchDisabledException("Pushes to this node have been administratively disabled."
```

Comment:
I don't think that this message is entirely better... `metadataStore.getReadOnlyFetchEnabledUnlocked()` will return false when a node is in `OFFLINE` state, right? This could be triggered automatically by BnP HA, so it does not reflect an "administrative" intervention. I think it would be confusing to state that in the error message.

Furthermore, I think it's useful to have the server state and node ID as part of the log message. You can still add the next line ("Please reach out to the cluster admin for assistance") but I think the beginning of the message should not be changed.